demo/IDWLayer.js

```diff
@@ -1,313 +0,0 @@
-/*
- * Based on, and a fork of, Bjoern Hoehrmann's OpenLayers Heatmap, copyright 
```

Comment:
I've moved this demo project to it's own repo at https://github.com/CUTR-at-USF/gtfsrdb-delay-demo, so it doesn't get lost.  Looks like it visualizes the delays on a map.